frontend/src/pages/ConnectionTest/ConnectionTest.tsx

```diff
@@ -234,8 +233,11 @@ function ApiTests(props: { connection: Connection }): JSX.Element {
     props.connection.api.on("KICK_NOTIFICATION", handleKickNotification);
     props.connection.api.on("PONG", handlePong);
     props.connection.api.on("FILTERS_DATA", handleFiltersData);
-    props.connection.api.on("GET_FILTERS_TEST_STATUS", handleGetFiltersTestStatus);
-    props.connection.api.on("FILTERS_TEST_STATUS", handleFiltersTestStatus);
+    props.connection.api.on(
+      "GET_FILTERS_DATA_SEND_TO_PARTICIPANT",
+      handleGetFiltersDataSendToParticipant
+    );
+    props.connection.api.on("FILTERS_DATA", handleFiltersData);
```

Comment:
This line is the same as line 235, is there a possibility for a typo? Could one of them be `props.connection.api.on("GET_FILTERS_DATA", handleGetFiltersData);`?